streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -396,28 +403,37 @@ public void shouldCloseProcessor() {
 
     @Test
     public void shouldThrowForUnknownTopic() {
-        final String unknownTopic = "unknownTopic";
-        final byte[] nullValue = null;
-
         testDriver = new TopologyTestDriver(new Topology(), config);
-        assertThrows(IllegalArgumentException.class, () -> {
-            testDriver.pipeRecord(unknownTopic, new TestRecord<byte[], byte[]>(nullValue), new ByteArraySerializer(), new ByteArraySerializer(), Instant.now());
-        });
+        assertThrows(
+            IllegalArgumentException.class,
+            () -> testDriver.pipeRecord(
+                "unknownTopic",
+                new TestRecord<>((byte[]) null),
+                new ByteArraySerializer(),
+                new ByteArraySerializer(),
+                Instant.now())
+        );
     }
 
     @Test
     public void shouldThrowForMissingTime() {
         testDriver = new TopologyTestDriver(new Topology(), config);
```

Comment:
This is not related to change in this PR, but after second read, should we setup non-empty topology here.
to prevent IllegalStateException to be thrown due to empty topology after some future change.
```suggestion
         testDriver = new TopologyTestDriver(setupSourceSinkTopology(), config);
```

Reply:
As above.